guides/common/modules/proc_configuring-pull-transport.adoc

```diff
@@ -0,0 +1,27 @@
+[id="configuring-pull-provider_{context}"]
+= Configuring Remote Execution for Pull Client
+
+By default, Remote Execution uses SSH as the transport mechanism for the Script provider.
+However, Remote Execution also offers a pull-based transport option, which communicates with hosts via MQTT.
+This is a good option if you are migrating from `katello-agent` which also uses pull-based communication.
+
+If you wish to use Remote Execution with the pull-based transport, run the following `{foreman-installer}` command on each {SmartProxy}:
+
+[options="nowrap" subs="quotes,attributes"]
+----
+
+ # {installer-scenario-smartproxy} \
+  --foreman-proxy-plugin-remote-execution-script-mode pull-mqtt
+
+----
+
+The mode is configured per {SmartProxy}.
+Some {SmartProxies} may be configured to use `pull-mqtt` mode while others use `ssh`.
+If this is the case, it is possible that one remote job on a given host will use the pull client, and the next job on the same host will use SSH.
+If you wish to avoid this scenario, configure all {SmartProxies} to use the same mode.
+
+After the pull-based transport is set up on each {SmartProxy}, it must also be configured on each host.
+See Configuring_and_Setting_up_Remote_Jobs_admin[Configuring and Setting Up Remote Jobs] in the _Managing Hosts_ guide.
+
+NOTE: The `pull-mqtt` mode works only with the Script provider.
+Ansible and other providers will continue to use SSH.
```

Comment:
*puts on pedantic hat*

This is somewhat misleading. Other providers I'm aware of are ansible and salt. Ansible uses ssh by default, but can be configured to use other transports per host by setting `ansible_connection` variable. Salt uses a transport based on zeromq by default.

Reply:
```
Ansible and other providers will continue to use their default transport settings.
```
Is that better?

Reply:
Sounds good